app/controllers/CyaMaintainProfileController.scala

```diff
@@ -60,7 +60,7 @@ class CyaMaintainProfileController @Inject() (
             HasNirmsSummary.rowUpdate(request.userAnswers)
           ).flatten
         )
-        Ok(view(list, routes.CyaMaintainProfileController.onSubmitNirms))
+        Ok(view(list, routes.CyaMaintainProfileController.onSubmitNirms, "hasNirms"))
```

Comment:
We could move `hasNirms, ukimsNumber and etc.` to `Utils/Constant`. That way, it's less error-prone. But no strong opinion on this, the rest looks good!